docs/src/content/components/datepicker.mdx

```diff
@@ -302,6 +302,12 @@ return (
     type="React.Dispatch<React.SetStateAction<ValueOf<typeof slides>>>"
     desc="the set state action for the selected date"
   />
+  <TypesProp
+    name="dateFormat"
+    optional
+    type="string"
+    desc="a <a href='https://date-fns.org/v2.16.1/docs/format'>date-fns compatible format string</a>"
+  />
```

Comment:
After running `yarn build` and `yarn start`, these additions don't show up in the documentation. Is there something else I need to do?